### ISSUE TITLE
dnsmasq binding to port 53

### ISSUE THREAD

**Issue by lbthomsen** [subutai-io/snap#239]
The fact that dnsmasq binds to port 53 causes problems when running on a number of host systems.  By default, Ubuntu Desktop is already running dnsmasq and many host systems may run bind already.

It must be possible to bind dnsmasq to a higher port - for example 5353 and then use an iptables port forward for all containers.

**Comment by akarasulu** [comment]
Yes this is a nice enhancement to avoid those headaches. I'd definitely do this especially with a shift to LXD.
